src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -15691,43 +15649,51 @@ class GetBookRequestTestOnSuccessMap
 
 // [START canonical]
 // FIXME: import everything this sample needs
-class PublishSeriesAsyncFlattenedAsyncPiVersion
+public class PublishSeriesAsyncFlattenedAsyncPiVersion
 {
     // [START canonical_core]
-   /// <summary>
-   /// Testing &lt;@calling forms&gt;
-   /// </summary>
-   /// <param name="shelfName">The name of the shelf where books are published to.</param>
-   /// <param name="edition">The edition of the series.</param>
-   public static void SamplePublishSeries(string shelfName, uint edition)
-   {
-       public static async Task SamplePublishSeriesAsync(string shelfName, uint edition)
-       {
-           LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
-           // string shelfName = "Math"
-           // uint edition = 123
-           Shelf shelf = new Shelf
-           {
-               Name = "Math",
-           };
-           scg::IEnumerable<Book> books = new scg::List<Book>();
-           uint edition = 123;
-           SeriesUuid seriesUuid = new SeriesUuid
-           {
-               SeriesString = "xyz3141592654",
-           };
-           stt::Task<PublishSeriesResponse> response = libraryServiceClient.PublishSeriesAsync(shelf, books, edition, seriesUuid);
-           // FIXME: inspect the results
-       }
-       // [END canonical_core]
-   }
-
-   // [END canonical]
-   public static void Main(String[] args)
-   {
-   	// FIXME: pass in commandline arguments
-   	// FIXME: call the sample function
-   }
+    /// <summary>
+    /// Testing &lt;@calling forms&gt;
+    /// </summary>
+    /// <param name="shelfName">The name of the shelf where books are published to.</param>
+    /// <param name="edition">The edition of the series.</param>
+    public static async Task SamplePublishSeriesAsync(string shelfName, uint edition)
+    {
+        LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
+        // string shelfName = "Math"
+        // uint edition = 123
+        Shelf shelf = new Shelf
+        {
+            Name = "Math",
+        };
+        scg::IEnumerable<Book> books = new scg::List<Book>();
```

Comment:
```suggestion
        IEnumerable<Book> books = new List<Book>();
```

Yuck, kill any of `foo::` that you see :D

Reply:
Yes, good point. In a sample this namespace aliasing is not desirable.

@hzyi-google Changing `CSharpAliasMode.Global` to `CSharpAliasMode.MessagesOnly` when instantiating the `CSharpSurfaceNamer` for sample-generation will fix this for all sample code.

Reply:
(Or the `scg::` may just be a literal within the snippit? I'm not sure...)